src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/Content/PageBody/_details.html.twig

```diff
@@ -112,16 +114,16 @@
                                                     {% set currencyCode = channel.baseCurrency.code %}
                                                     {% set price = channelPricing.price %}
                                                     {% set originalPrice = channelPricing.originalPrice ?? 0 %}
-                                                    {% set lowestPriceBeforeDiscount = channelPricing.lowestPriceBeforeDiscount ?? 0 %}
+                                                    {% set lowestPriceBeforeDiscount = channelPricing.lowestPriceBeforeDiscount %}
 
-                                                    <tr>
-                                                        <td>
-                                                            <span class="badge me-2" style="background-color: {{ channel.color }};"></span>
+                                                    <tr {{ sylius_test_html_attribute('variants', "%s"|format(variant.name) ) }}>
+                                                        <td {{ sylius_test_html_attribute('channel-name') }}>
```

Comment:
Why are we using names instead of codes here?

Reply:
Just to not force to convert it in behat tests, as we are passing names in the scenarios